lib/readline.js

```diff
@@ -429,7 +428,9 @@ Interface.prototype._tabComplete = function() {
           }
         }
         handleGroup(self, group, width, maxColumns);
+      }
 
+      {
```

Comment:
To summarize, if we use `var` then remove the block scope. If we keep the block scope then switch to `let`. Personally I don't care either way. @addaleax Let's do one or the other, but I'll leave that up to you.